Godeps/Godeps.json

```diff
@@ -1,6 +1,9 @@
 {
 	"ImportPath": "github.com/uber/ringpop-go",
 	"GoVersion": "go1.5.1",
+	"Packages": [
+		"./..."
```

Comment:
When you use `godeps`, you should save dependencies for all subpackages as well, and so you should run:
```bash
godep save ./...
```

It looks like it was previously only doing godeps for the root package.